### ISSUE TITLE
France code and others is seemingly incorrect

### ISSUE THREAD

**Issue by dicash** [ChristoPy/countries-phone-masks#19]
Must be 0x xx xx xx xx from https://en.wikipedia.org/wiki/Telephone_numbers_in_France.

I wonder how much more patterns are incorrect here...

**Comment by dicash** [comment]
Ok, random pick to ChatGPT:

```
Finland (+358): The format "(###)###-##-##" could be used, but it's more common to see Finnish numbers formatted differently depending on whether they're mobile or landline. For mobile phones, a more common format might be "+358 4# ### ####" or "+358 5# ### ####".

France (+33): The format "(###)###-###" is almost correct, but typically, French phone numbers are divided into sets of two digits when written down or spoken, so a format like "+33 # ## ## ## ##" would reflect the common grouping used in France.

French Guiana (+594): The mask "#####-####" is suitable for French Guiana. Being an overseas department of France, it shares the same numbering system as mainland France but has its own country code. A format reflecting the grouping into two digits like in France might also be applicable here, "+594 ### ## ## ##".

French Polynesia (+689): The format "##-##-##" is correct for French Polynesia. The phone numbers here are typically six digits long after the country code, and the suggested mask matches the common format.
```

So it seems library has own ideas about numbers formatting - I wonder what's the source of data?

**Comment by dicash** [comment]
Also: UM, EH countries are missing

**Comment by ChristoPy** [comment]
Hi! All the formats here are from gists and wikipedia articles.

As there are many, sometimes they are incorrect.
Could you help fixing these masks and adding those new ones?

Just open a PR and I can launch a new version right away 😄

**Comment by dicash** [comment]
I can't help contributing since working from not personal setup, but I can share more-less correct phone masks from ChatGPT:
```
{
  AF: ['+93', '0## ### ####'],
  AX: ['+358', '18 ### ####'],
  AL: ['+355', '(###) ### ###'],
  DZ: ['+213', '(###) ### ###'],
  AS: ['+1', '(684) ###-####'],
  AD: ['+376', '### ###'],
  AO: ['+244', '(###) ### ###'],
  AI: ['+1', '(264) ###-####'],
  AG: ['+1', '(268) ###-####'],
  AR: ['+54', '(###) ###-####'],
  AM: ['+374', '## ### ###'],
  AW: ['+297', '### ####'],
  AU: ['+61', '# #### ####'],
  AT: ['+43', '(###) ### ####'],
  AZ: ['+994', '(###) ### ## ##'],
  BS: ['+1', '(242) ###-####'],
  BH: ['+973', '#### ####'],
  BD: ['+880', '# ### ###'],
  BB: ['+1', '(246) ###-####'],
  BY: ['+375', '(##) ###-##-##'],
  BE: ['+32', '(###) ### ###'],
  BZ: ['+501', '###-####'],
  BJ: ['+229', '##-##-####'],
  BM: ['+1', '(441) ###-####'],
  BT: ['+975', '# ### ###'],
  BO: ['+591', '# ### ####'],
  BA: ['+387', '## ###-###'],
  BW: ['+267', '## ### ###'],
  BR: ['+55', '(##) ####-####'],
  IO: ['+246', '### ####'],
  BN: ['+673', '### ####'],
  BG: ['+359', '(###) ### ###'],
  BF: ['+226', '## ## ####'],
  BI: ['+257', '## ## ####'],
  KH: ['+855', '## ### ###'],
  CM: ['+237', '#### ####'],
  CA: ['+1', '(###) ###-####'],
  CV: ['+238', '(###) ## ##'],
  KY: ['+1', '(345) ###-####'],
  CF: ['+236', '## ## ####'],
  TD: ['+235', '## ## ## ##'],
  CL: ['+56', '# #### ####'],
  CN: ['+86', '(###) #### ####'],
  CX: ['+61', '3 ## ## ##'],
  CC: ['+61', '8 ## ## ##'],
  CO: ['+57', '(###) ### ####'],
  KM: ['+269', '## ## ## ##'],
  CG: ['+242', '## ### ####'],
  CK: ['+682', '## ###'],
  CR: ['+506', '#### ####'],
  HR: ['+385', '## #### ###'],
  CU: ['+53', '# ### ####'],
  CY: ['+357', '## ### ###'],
  CZ: ['+420', '### ### ###'],
  CD: ['+243', '### ### ###'],
  DK: ['+45', '## ## ## ##'],
  DJ: ['+253', '## ## ## ##'],
  DM: ['+1', '(767) ###-####'],
  DO: ['+1', '(###) ###-####'],
  EC: ['+593', '## ### ####'],
  EG: ['+20', '# #### ####'],
  SV: ['+503', '#### ####'],
  GQ: ['+240', '## ### ####'],
  ER: ['+291', '# ### ###'],
  EE: ['+372', '#### ####'],
  SZ: ['+268', '#### ####'],
  ET: ['+251', '## ### ####'],
  FK: ['+500', '#####'],
  FO: ['+298', '######'],
  FJ: ['+679', '### ####'],
  FI: ['+358', '## ### ####'],
  FR: ['+33', '## ## ## ## ##'],
  GF: ['+594', '### ## ## ##'],
  PF: ['+689', '## ## ##'],
  GA: ['+241', '# ## ## ##'],
  GM: ['+220', '### ####'],
  GE: ['+995', '(###) ## ## ##'],
  DE: ['+49', '(###) ### ####'],
  GH: ['+233', '## ### ####'],
  GI: ['+350', '### #####'],
  GR: ['+30', '(###) ### ####'],
  GL: ['+299', '## ## ##'],
  GD: ['+1', '(473) ###-####'],
  GP: ['+590', '### ## ## ##'],
  GU: ['+1', '(671) ###-####'],
  GT: ['+502', '# ### ####'],
  GG: ['+44', '(####) ### ###'],
  GN: ['+224', '## ### ###'],
  GW: ['+245', '# #######'],
  GY: ['+592', '### ####'],
  HT: ['+509', '## ## ####'],
  VA: ['+379', '### ####'],
  HN: ['+504', '####-####'],
  HK: ['+852', '#### ####'],
  HU: ['+36', '(##) ### ####'],
  IS: ['+354', '### ####'],
  IN: ['+91', '####### ###'],
  ID: ['+62', '(###) ###-####'],
  IR: ['+98', '(###) ### ####'],
  IQ: ['+964', '(###) ### ####'],
  IE: ['+353', '(###) ### ###'],
  IM: ['+44', '(####) ### ###'],
  IL: ['+972', '#-###-####'],
  IT: ['+39', '(###) #### ###'],
  CI: ['+225', '## ## ## ##'],
  JM: ['+1', '(876) ###-####'],
  JP: ['+81', '(###) ###-####'],
  JE: ['+44', '(####) ### ###'],
  JO: ['+962', '# #### ####'],
  KZ: ['+7', '(###) ###-##-##'],
  KE: ['+254', '### ######'],
  KI: ['+686', '## ###'],
  KP: ['+850', '###-###-####'],
  KR: ['+82', '(###) ####-####'],
  XK: ['+383', '## ### ###'],
  KW: ['+965', '#### ####'],
  KG: ['+996', '(###) ###-###'],
  LA: ['+856', '## ## ####'],
  LV: ['+371', '## ### ###'],
  LB: ['+961', '## ### ###'],
  LS: ['+266', '# ### ####'],
  LR: ['+231', '## ### ###'],
  LY: ['+218', '##-#######'],
  LI: ['+423', '(###) ###-####'],
  LT: ['+370', '(###) ## ###'],
  LU: ['+352', '### ### ###'],
  MO: ['+853', '#### ####'],
  MG: ['+261', '## ## ### ##'],
  MW: ['+265', '# #### ####'],
  MY: ['+60', '(###) ###-###'],
  MV: ['+960', '###-####'],
  ML: ['+223', '## ## ####'],
  MT: ['+356', '#### ####'],
  MH: ['+692', '###-####'],
  MQ: ['+596', '### ## ## ##'],
  MR: ['+222', '## ## ####'],
  MU: ['+230', '### ####'],
  YT: ['+262', '### ## ## ##'],
  MX: ['+52', '(###) ###-####'],
  FM: ['+691', '### ####'],
  MD: ['+373', '#### ####'],
  MC: ['+377', '(###) ###-###'],
  MN: ['+976', '## ## ####'],
  ME: ['+382', '## ### ###'],
  MS: ['+1', '(664) ###-####'],
  MA: ['+212', '#-####-####'],
  MZ: ['+258', '## ### ####'],
  MM: ['+95', '# ### ####'],
  NA: ['+264', '## ### ####'],
  NR: ['+674', '### ####'],
  NP: ['+977', '#-######'],
  NL: ['+31', '## ### ####'],
  NC: ['+687', '##.##.##'],
  NZ: ['+64', '## ### ###'],
  NI: ['+505', '#### ####'],
  NE: ['+227', '## ## ## ##'],
  NG: ['+234', '## ### ####'],
  NU: ['+683', '####'],
  NF: ['+672', '3## ###'],
  MK: ['+389', '## ### ###'],
  MP: ['+1', '(670) ###-####'],
  NO: ['+47', '### ## ###'],
  OM: ['+968', '#### ####'],
  PK: ['+92', '(###) #######'],
  PW: ['+680', '### ####'],
  PS: ['+970', '## ### ####'],
  PA: ['+507', '####-####'],
  PG: ['+675', '(###) ## ###'],
  PY: ['+595', '## ### ####'],
  PE: ['+51', '(###) ### ###'],
  PH: ['+63', '(###) ###-####'],
  PN: ['+870', '### ####'],
  PL: ['+48', '### ### ###'],
  PT: ['+351', '## ### ####'],
  PR: ['+1', '(###) ###-####'],
  QA: ['+974', '#### ####'],
  RE: ['+262', '##### ####'],
  RO: ['+40', '## ### ####'],
  RU: ['+7', '(###) ###-##-##'],
  RW: ['+250', '(###) ### ###'],
  BL: ['+590', '### ## ## ##'],
  SH: ['+290', '####'],
  KN: ['+1', '(869) ###-####'],
  LC: ['+1', '(758) ###-####'],
  MF: ['+590', '### ## ## ##'],
  PM: ['+508', '## ## ##'],
  VC: ['+1', '(784) ###-####'],
  WS: ['+685', '####'],
  SM: ['+378', '(####) ######'],
  ST: ['+239', '## ####'],
  SA: ['+966', '# ### ####'],
  SN: ['+221', '## ### ####'],
  RS: ['+381', '## ### ####'],
  SC: ['+248', '# ### ###'],
  SL: ['+232', '## ######'],
  SG: ['+65', '#### ####'],
  SX: ['+1', '(721) ###-####'],
  SK: ['+421', '(###) ### ###'],
  SI: ['+386', '## ### ###'],
  SB: ['+677', '### ####'],
  SO: ['+252', '# ### ###'],
  ZA: ['+27', '## ### ####'],
  GS: ['+500', '####'],
  SS: ['+211', '# ### ####'],
  ES: ['+34', '### ### ###'],
  LK: ['+94', '## ### ####'],
  SD: ['+249', '## ### ####'],
  SR: ['+597', '###-####'],
  SJ: ['+47', '### ## ###'],
  SE: ['+46', '## ### ####'],
  CH: ['+41', '## ### ####'],
  SY: ['+963', '## #### ###'],
  TW: ['+886', '#### ####'],
  TJ: ['+992', '## ### ####'],
  TZ: ['+255', '## ### ####'],
  TH: ['+66', '# #### ####'],
  TL: ['+670', '#### ####'],
  TG: ['+228', '## ## ## ##'],
  TK: ['+690', '####'],
  TO: ['+676', '####'],
  TT: ['+1', '(868) ###-####'],
  TN: ['+216', '## ### ###'],
  TR: ['+90', '(###) ### ####'],
  TM: ['+993', '# ### ###'],
  TC: ['+1', '(649) ###-####'],
  TV: ['+688', '####'],
  UG: ['+256', '## ### ####'],
  UA: ['+380', '## ### ####'],
  AE: ['+971', '# ### ####'],
  GB: ['+44', '#### ### ###'],
  US: ['+1', '(###) ###-####'],
  UM: ['+1', '(###) ###-####'],
  UY: ['+598', '# ### ## ##'],
  UZ: ['+998', '## ### ####'],
  VU: ['+678', '## ####'],
  VE: ['+58', '###-###-####'],
  VN: ['+84', '## #### ###'],
  VG: ['+1', '(284) ###-####'],
  VI: ['+1', '(340) ###-####'],
  WF: ['+681', '## ####'],
  EH: ['+212', '###-###-###'],
  YE: ['+967', '# ### ###'],
  ZM: ['+260', '## ### ####'],
  ZW: ['+263', '# #######'],
}
```

**Comment by ChristoPy** [comment]
Hey! Just fixed in 1.1.1 version fb0687af01cc42e1007e0e1061a6d1c8acc3e677
